<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>92</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Сделайте ход</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ChooseBoxLayout">
    <item>
     <widget class="QPushButton" name="rock_Button">
      <property name="text">
       <string>Камень</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="scissors_Button">
      <property name="text">
       <string>Ножницы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="paper_Button">
      <property name="text">
       <string>Бумага</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
